doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,99 @@
+.. raw:: html
 
-.. image:: perception_configuration_demo.png
-   :width: 700px
+        <video width="700px" controls="true" autoplay="true" loop="true">
+            <source src="../../../_static/videos/perception_pipeline_demo.webm" type="video/webm">
+            MoveIt Perception Pipeline Demo
+        </video>
 
 Getting Started
 ---------------
-If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
-
-Configuration
--------------
-
-In this section, we will walk through configuring the 3D sensors on your robot with MoveIt. The primary component in MoveIt that deals with 3D perception is the Occupancy Map Updater. The updater uses a plugin architecture to process different types of input. The currently available plugins in MoveIt are:
-
-* The PointCloud Occupancy Map Updater: which can take as input point clouds (``sensor_msgs/PointCloud2``)
-
-* The Depth Image Occupancy Map Updater: which can take as input Depth Images (``sensor_msgs/Image``)
-
-To use the Occupancy Map Updater, it is only necessary to set the appropriate parameters on the ROS parameter server and to call ``startWorldGeometryMonitor`` from your ``PlanningSceneMonitor``.  This latter step is performed automatically when using Move Group functionality like in this tutorial's example, so in that case it is only necessary to set the parameters for the octomap and octomap updater.
-
-YAML Configuration file (Point Cloud)
-+++++++++++++++++++++++++++++++++++++
-
-We will have to generate a YAML configuration file for configuring the 3D sensors. Please see :panda_codedir:`this example file<config/sensors_kinect_pointcloud.yaml>` for processing point clouds.
-
-Save this file in the config folder in the robot's moveit_config package with name "sensors_kinect_pointcloud.yaml": ::
-
- sensors:
-   - sensor_plugin: occupancy_map_monitor/PointCloudOctomapUpdater
-     point_cloud_topic: /camera/depth_registered/points
-     max_range: 5.0
-     point_subsample: 1
-     padding_offset: 0.1
-     padding_scale: 1.0
-     max_update_rate: 1.0
-     filtered_cloud_topic: filtered_cloud
-
-**The general parameters are:**
-
-* *sensor_plugin*: The name of the plugin that we are using.
-* *max_update_rate*: The octomap representation will be updated at rate less than or equal to this value.
-
-**Parameters specific to the Point cloud updater are:**
-
-* *point_cloud_topic*: This specifies the topic to listen on for a point cloud.
-
-* *max_range*: (in m) Points further than this will not be used.
-
-* *point_subsample*: Choose one of every *point_subsample* points.
-
-* *padding_offset*: The size of the padding (in cm).
-
-* *padding_scale*: The scale of the padding.
-
-* *filtered_cloud_topic*: The topic on which the filtered cloud will be published (mainly for debugging). The filtering cloud is the resultant cloud after self-filtering has been performed.
-
-
-YAML Configuration file (Depth Map)
```

Comment:
We can drop the depth map since it is not covered in this tutorial anyways I guess

Reply:
Added